web/src/lib/utils/tasksMarkdownReader.ts

```diff
@@ -29,6 +29,7 @@ export const convertAnytimeActionTaskMd = (
     filename,
     ...anytimeActionGrayMatter,
     category: newAnytimeActionCategoriesData,
+    type: "task",
```

Comment:
We were manually setting this in the `AnytimeActionDropdown` for every Anytime Action. I just moved this to the point that we read in the data so it's perpetually set. This function only works with Anytime Action Tasks, the Reinstatements are handled by a different function.